notebooks/demo_attribute_extraction.ipynb

```diff
@@ -53,7 +53,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "entity_model = spacy.load(\"en_ner_craft_md\")"
+    "entity_model = spacy.load(\"data_and_models/models/ner_er/model-chemical\")"
```

Comment:
What is this `ner_er/model-chemical`?
I couldn't find it tracked with `dvc` and I think I have not really seen it before...

Reply:
It's listed in the `data_and_models/models/ner_er/.gitignore` as well as in `data_and_models/pipelines/ner/params.yaml` so I assumed it's a legit model that we have trained.

Reply:
Sorry for jumping in out of the blue, `ner_er/model-XX` (XX being `entity type`) are the new models trained after:
- switching to `transformers` backbone 
- the decision of one model per entity type was taken.

Reply:
Ah thanks @EmilieDel! I had an older branch checked-out locally and couldn't see it.